docs/source/configuration/commandline.rst

```diff
@@ -99,7 +99,7 @@ Html report related
     Path to custom mustache template/css directory, needs to contain full set of templates.
     Bundled templates can be found in jar '/layout' and can be extracted with jar tool or any zip capable tool.
 [-maxdet number]
-    Limit for when tables should be shown with details.
+    Threshold for details in diagrams, increase if you still want details, checked against sum of tables and views.
```

Comment:
Make more concise.
Maybe something like "Maximum number of columns displayed in the entity-relationship diagram"

Possibly add default value and default behaviour

Reply:
Since your example is incorrect, there's something wrong with the text or the "option"

Show table/view details(all columns) as long as the sum(tables + views) are lower than this assigned value. Default is 400 i think.

Reply:
Yeah, this is definitely not obvious at all. In addition to the question of what `details` is, what `diagrams` are we talking about? Compact, large, one degree, two degrees?

I think I need a better understanding of the scenario where we use this, before I can suggest improvements.

Reply:
https://github.com/schemaspy/schemaspy/blob/07f5f074c908a68b598848e513693b487d3e3636/src/main/java/org/schemaspy/SchemaAnalyzer.java#L358